### PR TITLE
[CORE][DOC]Add event log conf.

### PR DESCRIPTION
**Pull request by guoxiaolongzte**
## What changes were proposed in this pull request?

Event Log Server has a total of five configuration parameters, and now the description of the other two configuration parameters on the doc, user-friendly access and use.

## How was this patch tested?

manual tests

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by guoxiaolongzte** [comment]
@srowen 
Help to review the code, thanks.

**Comment by guoxiaolongzte** [comment]
Why the administrator did not have time to review the code?

**Comment by guoxiaolongzte** [comment]
@HyukjinKwon @jerryshao
Help to review the code, thanks.

File: docs/configuration.md [hunk 1]
```diff
@@ -740,6 +740,20 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.eventLog.overwrite</code></td>
+  <td>true</td>
```
Comment (srowen) [inline docs/configuration.md]: The default appears to be false.

File: docs/configuration.md [hunk 2]
```diff
@@ -740,6 +740,20 @@ Apart from these, the following properties are also available, and may be useful
...
+  <td>
+    Whether to overwrite any existing files.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.eventLog.buffer.kb</code></td>
+  <td>100</td>
+  <td>
+    Buffer size to use when writing to output streams.
```
Comment (srowen) [inline docs/configuration.md]: Buffer size in KB

**Comment by guoxiaolongzte** [comment]
@srowen 
Help to review the code, thanks.

File: docs/configuration.md [hunk 3]
```diff
@@ -740,6 +740,20 @@ Apart from these, the following properties are also available, and may be useful
...
+  <td>false</td>
+  <td>
+    Whether to overwrite any existing files.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.eventLog.buffer.kb</code></td>
+  <td>100</td>
+  <td>
+    Buffer size to use when writing to output streams.Buffer size in KB.
```
Comment (srowen) [inline docs/configuration.md]: Might want to proofread these before pinging for review. You're missing a space before the second sentence, but, why not just: "Buffer size in KB to use when writing to output streams"

Comment (guoxiaolongzte) [reply docs/configuration.md]: I have fixed the description and correction unit.
Please check org.apache.spark.internal.config#EVENT_LOG_OUTPUT_BUFFER_SIZE

  private[spark] val EVENT_LOG_OUTPUT_BUFFER_SIZE = ConfigBuilder("spark.eventLog.buffer.kb")
    .bytesConf(ByteUnit.KiB)
    .createWithDefaultString("100k")

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**